### PR TITLE
removes feet voiceline from vox

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

removes feet voiceline from vox

## Why It's Good For The Game

i am going to fucking lose it if i hear some dumb fuck ai spam moth feet x50 in vox again. i am tired of breaking into rnd so i can upload laws to the ai to make it shut up 
and make it suffer the consequences of its actions with an equally shit law

the voiceline serves no meaningful purpose, it isnt helpful for describing anything, its used purely for the most basic, childish, simian vox announcements to elicit a base reaction from the entire crew and thus neuters actually interesting and funny usages of the vox system by encouraging people to spam the same thing over and over

tl;dr : it adds nothing and removes vox skill expression

## Changelog

:cl:
balance: removes feet voiceline from vox
/:cl:

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
Do we at least have meters if we're removing feet? (This is a joke)

Yeah I'm down for it

**Comment by carlarctg** [comment]
> Do we at least have meters if we're removing feet? (This is a joke)
> 
> Yeah I'm down for it

no but we do have a fahrenheit voiceline (worse than feet tbh)

**Comment by BlueMemesauce** [comment]
then they're just going to spam some other stupid thing
is feet worse than "coomer" or "abortions"
its pretty much just an admin issue

**Comment by carlarctg** [comment]
> then they're just going to spam some other stupid thing is feet worse than "coomer" or "abortions" its pretty much just an admin issue

i can remove those too (and also yes it is)

**Comment by LemonInTheDark** [comment]
if it's an admin issue why are we doing this

**Comment by carlarctg** [comment]
there is only so much admins can administrate. we can give everyone a 357 revolver for self defence only on penalty of ban but they eill still use it to shoot the clown. 

this adds nothing and at best causes an intractable admin issue

also, do we want 'do not use feet vox without a good reason' in silicon policy. do we really

**Comment by TheVekter** [comment]
Based.

I don't think this needs the "balance" tag tho.

**Comment by LemonInTheDark** [comment]
I am not interested in playing whackamole with the stupid shit people can say with vox until we run out of vox lines.
I do not think this is a problem you can solve with removal, am gonna talk to @tgstation/head-admins about how they feel about this stuff. I think this is an admin issue if spammed or just goofy dumb shit on its own (which is part of the charm of the game)

**Comment by LemonInTheDark** [comment]
> there is only so much admins can administrate. we can give everyone a 357 revolver for self defence only on penalty of ban but they eill still use it to shoot the clown.
> 
> this adds nothing and at best causes an intractable admin issue
> 
> also, do we want 'do not use feet vox without a good reason' in silicon policy. do we really

your example is bad and you should feel bad. policy exists to clarify existing rules not to enumerate all the different ways you can be told to stop being cringe.

**Comment by Blacklist897** [comment]
FEET ARE SAVED

**Comment by carlarctg** [comment]
tg has fallen

**Comment by optimumtact** [comment]
this is clearly an admin issue.
